rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -158,7 +270,18 @@ components:
         dev:
           value: '1.2.0-SNAPSHOT'
           summary: A development version
-
+    TransformId:
+      name: TransformId
+      in: path
+      description: >
+        The ID of the transform request.
+      required: true
+      schema:
+        type: string
+        minLength: 32
+        maxLength: 32
+        pattern: '^[0-9a-zA-Z]+$'
```

Comment:
❗️ Since this id is internal and nobody outside of transform should know or care about it, I would keep this just a string and not lock us into an implementation.